src/completion/completeStateNames.ts

```diff
@@ -76,21 +78,33 @@ function getListOfItems(node: PropertyASTNode, options?: ASLOptions): string[] {
 function getCompletionList(
     items: string[],
     replaceRange: Range,
-    shouldAddLeftQuote?: boolean,
-    shouldAddLeftSpace?: boolean,
-    shoudlAddTrailingComma?: boolean
+    languageId: string,
+    options: CompleteStateNameOptions
 ) {
+    const {
+        shouldAddLeftQuote,
+        shouldAddRightQuote,
+        shouldAddLeadingSpace,
+        shoudlAddTrailingComma
+    } = options
+
     const list: CompletionList = {
         isIncomplete: false,
         items: items.map(name => {
+            const shouldWrapStateNameInQuotes = languageId === 'asl-yaml' && isStateNameReservedYamlKeyword(name)
```

Comment:
i see the string 'asl-yaml' used in several places, can we create constants for this